canonical/ubuntu.com
# add more notes/details into a new issue template

## Done

- Added more notes and details to a new issue template so that reporters hopefully provide more context. This is done the same way as in canonical.com.

## QA

- Cannot be tested until merged. After it's merged create a new issue and check the template.

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
